tests/integration/Dockerfile.in

```diff
@@ -0,0 +1,25 @@
+# Copyright 2016 Google Inc. All rights reserved.
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+
+# http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+FROM ${STAGING_IMAGE}
+
+COPY . /app
+WORKDIR /app
+
+RUN pip install -r requirements.txt
+
+ENV GOOGLE_APPLICATION_CREDENTIALS=/app/auth.json
+
+ENTRYPOINT ["python"]
+CMD ["server.py"]
```

Comment:
Make sure to add it to `requirements.txt` (can't tell you how many times I've forgot this)